src/library/transform/scan.cpp

```diff
@@ -1,147 +1,64 @@
-/* ************************************************************************
```

Comment:
If this file is marked for death, it's not a big priority, but i also think it's easy to add at the same time as the other fixes.